3523/1/sorter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# This is Sorter class. This class sort gems by populatity and gives result
+class Sorter
+  attr_reader :result
+  def initialize(strings)
+    @strings = strings
+    @result = []
+  end
+
+  def result_sort
+    result = result_strings(sort(@strings))
+    result.reverse!
```

Comment:
instead of calling `reverse!` you can sort in right order directly in the `sort` method